docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md

```diff
@@ -16,7 +17,7 @@ Embedded Checkout lets you place BigCommerce’s Optimized One-Page checkout ont
 This article assumes you have familiarity with the following concepts:
 
 * Creating and managing a server-side application
-* Making and recieving API calls from within your app
+* Making and receiving API calls from within your app
 * Using your application to make changes to a front end
 
 ### Prerequisites
```

Comment:
Remove the parenthetical content in the following sentence and put it beneath the unordered list contents.

**Suggestion:** 
"For more information, see [OAuth Scopes](https://developer.bigcommerce.com/api-docs/getting-started/authentication/rest-api-authentication#oauth-scopes).